api/src/main/java/org/eclipse/microprofile/config/spi/ConfigSourceInterceptorContext.java

```diff
@@ -0,0 +1,9 @@
+package org.eclipse.microprofile.config.spi;
+
+public interface ConfigSourceInterceptorContext {
+    String getKey();
+
+    Class getConfigSource();
+
+    String proceed(String key) throws Exception;
```

Comment:
Since `ConfigSourceInterceptor` would return a `ConfigValue`, so should this method.  Also I'd drop the `throws Exception` since we only want interceptors to throw the spec-defined exception types (`IllegalArgument` for validation problems, `NoSuchElement` for missing value cases).

Reply:
Ops, I did create the class but forgot to use it in the return for this method.